### DISCUSSION TITLE
Integration broken ?

### DISCUSSION THREAD

**Discussion by MaximeNagel** [DurgNomis-drol/ha_toyota#200] [General]
Hey everyone,

I've been using this integration for a while now with my 2021 Toyota Corolla.

Back in November, if I remember correctly, Toyota switched to a new app and API, at least that's what I gathered from various posts on issues. The integration broke for a few days, but thanks to CM000n's awesome work, I switched to the master branch and everything seemed to work almost normally again.

However, for the past few days, I've noticed that the integration isn't initializing anymore. I don't see any errors in the HA logs, but I get this message: "action is not supported by this vehicle" when I hover over the integration in HA.

![image](https://github.com/DurgNomis-drol/ha_toyota/assets/6407540/d73a4f32-436f-40f1-832a-ed10eed84d12)

Is the problem on my end, or is it just because you guys are doing a huge job integrating Toyota's changes?

Also, I want to take this opportunity to thank you for all the hard work. I know you're working without documentation, and that must be really tough!

Sorry for my english ;) French here !

**Comment by CM000n** [comment]
Duplicate. Please look at: https://github.com/DurgNomis-drol/ha_toyota/issues/160 😉

**Reply by MaximeNagel** [reply]
Ho sorry.... I did see this issue before sending my message but as the error was not the same as me, I was not sure if it was this problem.
Thanks for you quick answer !
